services/myget/myget.service.js

```diff
@@ -8,51 +9,73 @@ const { NugetVersionService: Version, NugetDownloadService: Downloads } =
   })
 
 class MyGetVersionService extends Version {
-  static examples = [
-    {
-      title: 'MyGet',
-      pattern: 'myget/:feed/v/:packageName',
-      namedParams: { feed: 'mongodb', packageName: 'MongoDB.Driver.Core' },
-      staticPreview: this.render({ version: '2.6.1' }),
-    },
-    {
-      title: 'MyGet (with prereleases)',
-      pattern: 'myget/:feed/vpre/:packageName',
-      namedParams: { feed: 'mongodb', packageName: 'MongoDB.Driver.Core' },
-      staticPreview: this.render({ version: '2.7.0-beta0001' }),
+  static openApi = {
+    '/myget/{feed}/{variant}/{packageName}': {
+      get: {
+        summary: 'MyGet Version',
+        parameters: pathParams(
+          { name: 'feed', example: 'mongodb' },
+          {
+            name: 'variant',
+            example: 'v',
+            schema: { type: 'variant', enum: ['v', 'vpre'] },
+            description:
+              'Latest stable version (`v`) or Latest version including prereleases (`vpre`).',
+          },
+          { name: 'packageName', example: 'MongoDB.Driver.Core' },
+        ),
+      },
     },
-    {
-      title: 'MyGet tenant',
-      pattern: ':tenant.myget/:feed/v/:packageName',
-      namedParams: {
-        tenant: 'tizen',
-        feed: 'dotnet',
-        packageName: 'Tizen.NET',
+    '/{tenant}/{feed}/{variant}/{packageName}': {
```

Comment:
In OpenAPI, the separator between params and another param or static part of the route has to be a `/` so we can't hard-code the `.myget` bit by making this `/{tenant}.myget/{feed}`, so I've just done the best I can here. This does lead to a slightly weird situation where if you put something in here that doesn't end with `.myget` you can put another service name in as the first param.

In general, the routes for this service are not good routes.

They still use the old `v`/`vpre` instead of an `include_prereleases` flag

Also the order of the params doesn't conform to our own guidance on how to correctly structure routes
https://github.com/badges/shields/blob/master/doc/badge-urls.md

Having done the best I can with what we've got, I'm going to suggest we just roll with this as it stands for the purposes of this migration and I'll raise an issue to bring the myget routes into line with current conventions and set up redirects.